src/vctrs.h

```diff
@@ -265,6 +265,38 @@ void hash_fill(uint32_t* p, R_len_t n, SEXP x);
 
 bool duplicated_any(SEXP names);
 
+// Missing values -----------------------------------------------
+
+// IEEE 754 defines binary64 as
```

Comment:
More precisely:

Annex F of C99 specifies that `double` should conform to the IEEE 754 type `binary64`, which is defined as: